feature/experimental/p4rt/ate_tests/performance_test/performance_test.go

```diff
@@ -0,0 +1,946 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package performance_test
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"sort"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/openconfig/featureprofiles/feature/experimental/p4rt/internal/p4rtutils"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+)
+
+var (
```

Comment:
I think everything but the flags in this var block can be const. Please move them into a new const block.

Reply:
Some of them have structs based on var for below . So i couldnt move them into const. Rest of them moved into const.

        ttl1                = uint8(1)
	hopLimit1     = uint8(1)
	lldpEthertype = uint32(0x88cc)
	gdpEtherType  = uint32(0x6007)